packages/api-client-next/src/defaultHeaders.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Source: https://github.com/shopware/shopware/blob/trunk/src/Core/PlatformRequest.php#L16
+ */
+type RequestHeaderName =
+  | "sw-context-token"
+  | "sw-access-key"
+  | "sw-language-id"
+  | "sw-currency-id"
+  | "sw-inheritance"
+  | "sw-version-id"
+  | "sw-include-seo-urls"
+  | "sw-skip-trigger-flow"
+  | "sw-app-integration-id"
+  | "indexing-behavior"
+  | "indexing-skip";
+
+export type ClientHeaders = Record<RequestHeaderName | string, string>;
```

Comment:
I like the way we provide built-in Shopware headers 👍🏼 

Also, I would like to have consistent way of building request headers as `ofetch` is build upon. So my proposition is to combine https://developer.mozilla.org/en-US/docs/Web/API/Headers/Headers constructor together with the headers that Shopware 6 supports (union) - you already defined in `RequestHeaderName` type.

thanks to this, working with header will be more natural for not only ofetch users, but fetch itself - that's a standard supported by all browsers at the moment.

Reply:
That could be the next step to support whole HeadersInit object (that could be also a tuple) so I went for a simpler object at this time and we can see if the usecases will require extending that

Reply:
alright, time will tell 👍🏼